src/Notify/Twig/NotifyExtension.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Notify\Twig;
+
+use Symfony\Component\Mercure\HubInterface;
+use Symfony\WebpackEncoreBundle\Twig\StimulusTwigExtension;
+use Twig\Environment;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * @author Mathias Arlaud <mathias.arlaud@gmail.com>
+ *
+ * @experimental
+ */
+final class NotifyExtension extends AbstractExtension
+{
+    private $hub;
+    private $stimulusTwigExtension;
+
+    public function __construct(HubInterface $hub, StimulusTwigExtension $stimulusTwigExtension)
+    {
+        $this->hub = $hub;
+        $this->stimulusTwigExtension = $stimulusTwigExtension;
+    }
+
+    public function getFunctions(): iterable
+    {
+        yield new TwigFunction('stream_notifications', [$this, 'renderStreamNotifications'], ['needs_environment' => true, 'is_safe' => ['html']]);
```

Comment:
let's leverage PHP 8.1+ first-class callables

```suggestion
        yield new TwigFunction('stream_notifications', $this->renderStreamNotifications(...), ['needs_environment' => true, 'is_safe' => ['html']]);
```

Reply:
Should we? I'd like to keep php 8.0 compat at least